.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -578,14 +580,14 @@ stages:
       displayName: Download MSI
       inputs:
         artifact: windows-msi-$(targetPlatform)
-        patterns: '**/*.msi'
+        patterns: '**/*profiler-beta.msi'
```

Comment:
TBH, this worries me slightly. With this change, we're no-longer testing the MSI in the pipeline at all. My preferred approach would be:

- Set this pattern based on the presence of the `USE_NATIVE_LOADER` variable.
- Add a new variable "IS_NATIVE_LOADER_PIPELINE". If this is set, only run the relevant stages (windows build + test)
- Set up a parallel pipeline that uses the same file, but which sets the `IS_NATIVE_LOADER_PIPELINE` variable.

This is the approach we used to test NGEN support in parallel. The downside is that it adds some complexity to the ultimate-pipeline.yml conditions, and it uses more runners, but I think it's preferable to still be testing the configuration we're shipping to customers.

Reply:
I can have a look.

But TBH, we won't ship this configuration to the customer so...

Reply:
> The plan being "next version only one MSI" (discussed on Tuesday)

That's the crucial info I missed, based on that, my suggestion is probably not necessary 🙂

Reply:
I don't recall we said the next release will have only one MSI (but it was the day I was completely jet lagged so maybe I missed it). I think it's safer for sure but there are many reasons why we may need to do a release. We have not been super precise on how long we want the tests to run with the native loader but I assumed that would be around a week. So we can push for a patch if anything rather urgent comes up meanwhile.